build/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+
+version: "2.4"
+
+services:
+
+  rv_builder:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        use_remote : ${use_remote}
+    image: rv_builder
+    container_name: rv_builder
+    restart: on-failure:5
+    # Mapping to the user workspace of 'sdouser' as created in the Dockerfile.
+    # If the username is updated in the Dockerfile, please replace 'sdouser' with the same.
+    volumes:
+      - ./../../rendezvous-service:/home/sdouser/rendezvous-service:rw
+    environment:
+      - http_proxy=${http_proxy}
+      - https_proxy=${https_proxy}
+      - no_proxy=${no_proxy}
+      - use_remote=${use_remote}
+
+    mem_limit: 2000m
+    mem_reservation: 1000m
```

Comment:
The mem_limit and mem_reservation values are very high. Suggest to start the container, execute `docker stats` command and based on that decide the values for mem_limit and mem_reservation fields.

Reply:
1. Reducing the mem limit to 1.5Gb. The value is set this high as the compilation is a memory-intensive task.